### DISCUSSION TITLE
[HELP] not really an ISSUE but more of a QUESTION

### DISCUSSION THREAD

**Discussion by keanugithub** [linuxserver/Heimdall#980] [General]
if i may, where can i find the HTML file of heimdall's dashboard? I want to edit the home page to add this
`<!--OST Widget code start--><div style="text-align:right;"><div style="min-height:360px;"><div style="width:100%;height:0;padding-bottom:50%;position:relative;"><iframe style="border:none;position:absolute;top:0;left:0;width:100%;height:100%;min-height:360px;border:none;overflow:hidden !important;" src="//openspeedtest.com/Get-widget.php"></iframe></div></div>Provided by <a href="https://openspeedtest.com">OpenSpeedtest.com</a></div><!-- OST Widget code end -->`

![image](https://user-images.githubusercontent.com/98110531/191111384-fc2d8e3c-f964-48ca-abce-a9ea7b19181d.png)

Anyone knows where or if there is another way to do it? Thank you appreciate the help!

**Comment by KodeStar** [comment]
you could try adding it in the Custom JS section in settings
